test/unit/DataRequirementHelpers.test.ts

```diff
@@ -625,4 +627,65 @@ describe('DataRequirementHelpers', () => {
       }
     });
   });
+
+  describe('extractDataRequirementsMeasurementPeriod', () => {
+    test('returns an empty object when no measurement info provided', () => {
+      expect(DataRequirementHelpers.extractDataRequirementsMeasurementPeriod({}, {})).toEqual({});
+    });
+
+    test('uses measurement period from options when both provided', () => {
+      const start = '2019-01-01';
+      const end = '2020-01-01';
+      const startCql = DateTime.fromJSDate(moment.utc(start, 'YYYYMDDHHmm').toDate(), 0);
+      const endCql = DateTime.fromJSDate(moment.utc(end, 'YYYYMDDHHmm').toDate(), 0);
+      const options: CalculationOptions = { measurementPeriodStart: start, measurementPeriodEnd: end };
+      const mp: fhir4.Period = { start: '2000-01-01', end: '2001-01-01' };
+      expect(DataRequirementHelpers.extractDataRequirementsMeasurementPeriod(options, mp)).toEqual({
+        'Measurement Period': new Interval(startCql, endCql)
+      });
+    });
+  });
+
+  test('uses measurement period from Measure when options not provided', () => {
+    const start = '2019-01-01';
+    const end = '2020-01-01';
+    const startCql = DateTime.fromJSDate(moment.utc(start, 'YYYYMDDHHmm').toDate(), 0);
+    const endCql = DateTime.fromJSDate(moment.utc(end, 'YYYYMDDHHmm').toDate(), 0);
+    const mp: fhir4.Period = { start, end };
+
+    expect(DataRequirementHelpers.extractDataRequirementsMeasurementPeriod({}, mp)).toEqual({
+      'Measurement Period': new Interval(startCql, endCql)
+    });
+  });
+
+  describe('createIntervalFromEndpoints', () => {
+    test('returns interval from start to end if both provided', () => {
+      const start = '2019-01-01';
+      const end = '2022-01-01';
+      const startCql = DateTime.fromJSDate(moment.utc(start, 'YYYYMDDHHmm').toDate(), 0);
+      const endCql = DateTime.fromJSDate(moment.utc(end, 'YYYYMDDHHmm').toDate(), 0);
+
+      expect(DataRequirementHelpers.createIntervalFromEndpoints(start, end)).toEqual(new Interval(startCql, endCql));
+    });
+
+    test('returns interval from start with duration 1 year if end not provided provided', () => {
+      const start = '2019-01-01';
+      const expectedEnd = '2020-01-01';
+      const startCql = DateTime.fromJSDate(moment.utc(start, 'YYYYMDDHHmm').toDate(), 0);
+      const endCql = DateTime.fromJSDate(moment.utc(expectedEnd, 'YYYYMDDHHmm').toDate(), 0);
+
+      expect(DataRequirementHelpers.createIntervalFromEndpoints(start)).toEqual(new Interval(startCql, endCql));
```

Comment:
```suggestion
      expect(DataRequirementHelpers.createIntervalFromEndpoints(start, undefined)).toEqual(new Interval(startCql, endCql));
```
Thoughts? Just makes things more clear but idk